### PR TITLE
cdc: add software and hardware recommendation

### PR DESCRIPTION
**Pull request by overvenus**
### What is changed, added or deleted? (Required)

Add software and hardware recommendation for TiCDC.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: N/A
- Other reference link(s): N/A

### CODE REVIEW

File: hardware-and-software-requirements.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ TiDB 支持部署和运行在 Intel x86-64 架构的 64 位通用硬件服务器
 | PD | 4 核+ | 8 GB+ | SAS, 200 GB+ | 千兆网卡 | 1（可与 TiDB 同机器） |
 | TiKV | 8 核+ | 32 GB+ | SSD, 200 GB+ | 千兆网卡 | 3 |
 | TiFlash | 32 核+ | 64 GB+ | SSD, 200 GB+ | 千兆网卡 | 1 |
+| TiCDC | 8 核+ | 16 GB+ | SSD, 200 GB+ | 千兆网卡 | 1 |
```
Comment (amyangfei) [inline hardware-and-software-requirements.md:58]: Is `SAS` enough here?

Comment (overvenus) [reply hardware-and-software-requirements.md:58]: It should be enough if we not enable file sort.
```suggestion
| TiCDC | 8 核+ | 16 GB+ | SAS, 200 GB+ | 千兆网卡 | 1 |
```

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -10,6 +10,21 @@ aliases: ['/docs-cn/dev/reference/tools/ticdc/manage/','/docs-cn/dev/reference/t
...
+TiCDC 生产环境软件和硬件推荐如下：
+
+| Linux 操作系统平台       | 版本         |
+| :----------------------- | :----------: |
+| Red Hat Enterprise Linux | 7.3 及以上   |
+| CentOS                   | 7.3 及以上   |
+
+| **CPU** | **内存** | **硬盘类型** | **网络** | **实例数量(最低要求)** |
+| --- | --- | --- | --- | --- |
+| 16 核+ | 64 GB+ | SSD | 万兆网卡（2 块最佳） | 2 |
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:24]:
```suggestion
| 16 核以上 | 64 GB 以上 | SSD | 万兆网卡（2 块最佳） | 2 |
```

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -10,6 +10,21 @@ aliases: ['/docs-cn/dev/reference/tools/ticdc/manage/','/docs-cn/dev/reference/t
...
+
+更多信息参见[TiDB 软件和硬件环境建议配置](/hardware-and-software-requirements.md)
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:26]:
```suggestion
更多信息参见 [TiDB 软件和硬件环境建议配置](/hardware-and-software-requirements.md)
```

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -10,6 +10,21 @@ aliases: ['/docs-cn/dev/reference/tools/ticdc/manage/','/docs-cn/dev/reference/t
 
 ## TiCDC 部署
 
+### 软件和硬件环境建议配置
+
+TiCDC 生产环境软件和硬件推荐如下：
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:15]:
```suggestion
在生产环境中，TiCDC 的软件和硬件配置推荐如下：
```

File: ticdc/manage-ticdc.md [hunk 5]
```diff
@@ -10,6 +10,21 @@ aliases: ['/docs-cn/dev/reference/tools/ticdc/manage/','/docs-cn/dev/reference/t
 
 ## TiCDC 部署
 
+### 软件和硬件环境建议配置
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:13]:
```suggestion
### 软件和硬件环境推荐配置
```

File: ticdc/manage-ticdc.md [hunk 6]
```diff
@@ -10,6 +10,21 @@ aliases: ['/docs-cn/dev/reference/tools/ticdc/manage/','/docs-cn/dev/reference/t
...
+在生产环境中，TiCDC 的软件和硬件配置推荐如下：
+
+| Linux 操作系统平台       | 版本         |
+| :----------------------- | :----------: |
+| Red Hat Enterprise Linux | 7.3 及以上   |
+| CentOS                   | 7.3 及以上   |
+
+| **CPU** | **内存** | **硬盘类型** | **网络** | **实例数量(最低要求)** |
+| --- | --- | --- | --- | --- |
+| 16 核以上 | 64 GB 以上 | SSD | 万兆网卡（2 块最佳） | 2 |
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:24]:
```suggestion
| 16 核+ | 64 GB+ | SSD | 万兆网卡（2 块最佳） | 2 |
```

**Pull request merged**